src/SmartComponents/BaselinesPage/EditBaseline/EditBaselineNameModal/EditBaselineNameModal.js

```diff
@@ -101,6 +101,18 @@ export class EditBaselineNameModal extends Component {
                     </Button>
                 ] }
             >
+                { error.status
+                    ? <Alert
+                        variant='danger'
+                        isInline
+                        title={ 'Status: ' + error.status }
+                    >
+                        <p>
+                            { error.detail }
+                        </p>
+                    </Alert>
+                    : <div></div>
+                }
```

Comment:
This could be written as just:

```js
{ error.status && <Alert
  variant='danger'
  isInline
  title={ 'Status: ' + error.status }>
    <p>
      { error.detail }
    </p>
</Alert> }
```